File: 백준/Bronze/31775. 글로벌 포닉스/글로벌 포닉스.py
a = input()
b = input()
c = input()

arr = []

if a[0] not in arr and (a[0] == 'l' or a[0] == 'k' or a[0] == 'p'):
    arr.append(a[0])
if b[0] not in arr and (b[0] == 'l' or b[0] == 'k' or b[0] == 'p'):
    arr.append(b[0])
if c[0] not in arr and (c[0] == 'l' or c[0] == 'k' or c[0] == 'p'):
    arr.append(c[0])

if 'l' in arr and 'k' in arr and 'p' in arr:
    print("GLOBAL")
else:
    print("PONIX")

File: 백준/Bronze/31775. 글로벌 포닉스/글로벌 포닉스.c
#include <stdio.h>

int main(void) { 
   char a[21], b[21], c[21];
   scanf("%s", a);
   scanf("%s", b);
   scanf("%s", c);

   if((a[0] != b[0]) && (a[0] != c[0]) && (b[0] != c[0])){
      if(a[0] == 'l' || a[0] == 'k' || a[0] == 'p'){
         if(b[0] == 'l' || b[0] == 'k' || b[0] == 'p'){
            if(c[0] == 'l' || c[0] == 'k' || c[0] == 'p'){
               printf("GLOBAL");
               return 0;
            }
         }
      }
   }

   printf("PONIX");

   return 0;  
   
}